src/Purebred.hs

```diff
@@ -28,15 +28,15 @@ import Purebred
 
 scrollKeybindings :: ('Scrollable' w) => ['Keybinding' v w]
 scrollKeybindings =
-  [ 'Keybinding' (EvKey (KChar 'j') []) ('scrollDown' ``chain`` 'continue')
-  , Keybinding (EvKey (KChar 'k') []) ('scrollUp' \`chain\` continue)
-  , Keybinding (EvKey (KChar 'd') []) ('scrollPageDown' \`chain\` continue)
-  , Keybinding (EvKey (KChar 'u') []) ('scrollPageUp' \`chain\` continue)
+  [ 'Keybinding' (EvKey (KChar 'j') []) ('scrollDown' \`chain\` 'continue')
+  , Keybinding (EvKey (KChar 'k') []) ('scrollUp' \`chain\` `continue`)
+  , Keybinding (EvKey (KChar 'd') []) ('scrollPageDown' \`chain\` `continue`)
+  , Keybinding (EvKey (KChar 'u') []) ('scrollPageUp' \`chain\` `continue`)
```

Comment:
Are backticks around `continue` correct?

Reply:
Hmm you're right. Thanks for spotting!